api/valor_api/backend/core/evaluation.py

```diff
@@ -320,15 +320,10 @@ def _create_responses(
 
         match parameters.task_type:
             case enums.TaskType.CLASSIFICATION:
-                missing_pred_keys, ignored_pred_keys = core.get_disjoint_keys(
-                    db,
-                    datum_filter,
-                    model_filter,
-                    label_map=parameters.label_map,
-                )
+                # classification tasks have to have matching sets of prediction label keys and ground truth label keys
                 kwargs = {
-                    "missing_pred_keys": missing_pred_keys,
-                    "ignored_pred_keys": ignored_pred_keys,
+                    "missing_pred_keys": [],
+                    "ignored_pred_keys": [],
```

Comment:
@ekorman are you happy with this change?

Reply:
these lists will necessarily be empty now that we're guaranteeing that the prediction label key set matches the ground truth label key set for all datums

Reply:
so should we not return these anymore?